### ISSUE TITLE
Styling React datapicker.

### ISSUE THREAD

**Issue by KowalczykBartek** [SAP/fundamental#381]
Hi Fundamental UI team !

as we discussed on Slack, we need help with datapicker styling. @saadmhybris knows location of our stash project and branch (I am not sure if I can share it on public github)

P.S @iketari wanted to be pinged as well :) 

Thanks.

**Comment by LeoT7508** [comment]
@KowalczykBartek 

This is what we have so far for date picker:
https://app.zeplin.io/project/5a8c458b6e231fc65bddbdfb/dashboard?seid=5aff064bfdf1003421ee2d28

**Comment by KowalczykBartek** [comment]
```
Please let me know if you guys have picked a datepicker
```
we implemented datapicker (as I mentioned on slack, in our stash repo, we have dedicated branch for this, I shared this link also). 

```
you can see the suggested design in the link leo posted above
```
I don't understand honestly. As we discussed, we was responsible for implementation of datapicker itself in react (because there is no datapicker in fundamental-ui created in plain-js) and then, fundamental-ui team based on our changes will add into fundamental-core, common styling for datapickers.

I am correct ?

**Comment by KowalczykBartek** [comment]
I will send you our datapicker and all related links in slack priv message.

**Comment by xak** [comment]
Quick update ... the `calendar` is completed and merged to `develop` but not yet released. The `date-picker` input is in progress.

You can see the SASS at https://github.com/SAP/fundamental/blob/develop/scss/components/calendar.scss and if you run the test project locally you can find it documented at http://localhost:3030/calendar

As promised, each element and date states are available as classes. So if you had a library that did not allow you to change the classes but maybe used `.date` and `.disabled-date` classes, then you could create a SASS file for that component and extend to those classes.

```
@import "~fundamental-ui/scss/components/calendar";

.third-party-calendar {
  .date {
    @extend %fd-calendar-item;
  }
  .disabled-date {
    @extend %fd-calendar-item-disabled;
  }
}
```

This would apply our styling to the third-party component. There's probably some fine-tuning we can do but it should work. Always open to feedback and ideas.

**Comment by KowalczykBartek** [comment]
Hi, 
any progress here ? can we expect that data-picker will appear in some near release ?

**Comment by LeoT7508** [comment]
@saadmhybris 

Any info about the date picker?

**Comment by saadmhybris** [comment]
@KowalczykBartek @LeoT7508 

both date and time picker are part of our next milestone - https://github.com/SAP/fundamental/milestone/10

**Comment by KowalczykBartek** [comment]
Thanks !

**Comment by saadmhybris** [comment]
@KowalczykBartek - we have released [v1.1.0](https://github.com/SAP/fundamental/releases/tag/v1.1.0) and this release includes styles for [calendar](https://sap.github.io/fundamental/components/calendar.html), [date picker](https://sap.github.io/fundamental/components/date-picker.html), [time](https://sap.github.io/fundamental/components/time.html), [time picker](https://sap.github.io/fundamental/components/time-picker.html). Please have a look and let us know if you need any further help.

**Comment by iketari** [comment]
@saadmhybris Looks great! Thank you!
